commands.go

```diff
@@ -29,10 +29,6 @@ type AppendEntriesResponse struct {
 
 	// We may not succeed if we have a conflicting entry
 	Success bool
-
-	// There are scenarios where this request didn't succeed
-	// but there's no need to wait/back-off the next attempt.
-	NoRetryBackoff bool
```

Comment:
How important is backward compatibility? If providing backward compatibility, perhaps it would be better to retain this field but document that it's deprecated and ignored.